README.md

```diff
@@ -142,7 +141,7 @@ GraphQL.js is [MIT-licensed](./LICENSE).
 
 ### Credits
 
-The `*.d.ts` files in this project are based on [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/54712a7e28090c5b1253b746d1878003c954f3ff/types/graphql) definitions written by:
+The `*.d.ts` files in this project are based on [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/54712a7e28090c5b1253b746d1878003c954f3ff/types/graphql). Definitions written by:
```

Comment:
issue: This changes the original meaning of the sentence.

"DefinitelyTyped" describes what definitions the files are based on.

Reply:
Whoops read this differently first time. Fixed